### PR TITLE
Set up for custom SQL being run for new wikis

### PR DESCRIPTION
**Pull request by kemayo**
This is a quick sketch of what'd probably work. I don't have patchdemo set up locally to test it properly.

### CODE REVIEW

**Comment by kemayo** [comment]
In theory the sql-perwiki directory would come to contain things like the interwiki table config, per @jdlrobson.

File: new/postinstall.sh [hunk 1]
```diff
@@ -17,6 +17,12 @@ php $PATCHDEMO/wikis/$NAME/w/maintenance/createAndPromote.php Mallory patchdemo1
 # This command may fail as --disable-autoblock was only added in 1.36, so suppress errors
 echo "Mallory" | php $PATCHDEMO/wikis/$NAME/w/maintenance/blockUsers.php --reason "Blocking account for testing" --disable-autoblock || echo "Can't block Mallory"
 
+# run arbitrary SQL
+for sql in $(find $PATCHDEMO/sql-perwiki -name "*.sql" -not -type d -printf '%P\n')
+do
+	mysql -u patchdemo -ppatchdemo patchdemo < $PATCHDEMO/sql-perwiki/$sql
```
Comment (edg2s) [inline new/postinstall.sh:23]: The database name for the wiki is patchdemo_$NAME

**Review by edg2s — CHANGES_REQUESTED** [review]

File: new/postinstall.sh (hunk 1 above)
Comment (kemayo) [reply new/postinstall.sh]: Fixed

**Review by edg2s — APPROVED** [review]
Tested locally, appears to work.

**Pull request merged**